tasks/main-tang.yml

```
@@ -26,6 +26,10 @@
   when: (nbde_server_fetch_keys | bool) or (nbde_server_deploy_keys | bool)
   include_tasks: tang-key-management.yml
 
+- name: Open port in firewalld
+  when: (nbde_server_port != 80) or (nbde_server_firewall_zone != "public")
```

```suggestion
  when: (nbde_server_port | int != 80) or (nbde_server_firewall_zone != "public")
```
This should fix the error from the log:
```
ERROR: The following numeric operations found at runtime need int or float cast filters:
[nbde_server_port] [ne] [80] at tasks/main-tang.yml:29
```
the issue is that since `nbde_server_port` is a public variable, there is no guarantee that the data type will be `int` e.g. if the user does `ansible-playbook -e nbde_server_port=80 ...` then the variable will be a `string` data type and the comparison `(nbde_server_port != 80)` will always fail.  Using `| int` to "cast" the type will ensure that the comparison is done correctly, and will also ensure that an error is generated if the data type cannot be cast to `int`.